src/bpmncwpverify/antlr/State.interp

```diff
@@ -24,10 +34,13 @@ WS
\ No newline at end of file
+[4, 1, 14, 70, 2, 0, 7, 0, 2, 1, 7, 1, 2, 2, 7, 2, 2, 3, 7, 3, 2, 4, 7, 4, 2, 5, 7, 5, 2, 6, 7, 6, 1, 0, 5, 0, 16, 8, 0, 10, 0, 12, 0, 19, 9, 0, 1, 0, 5, 0, 22, 8, 0, 10, 0, 12, 0, 25, 9, 0, 1, 0, 4, 0, 28, 8, 0, 11, 0, 12, 0, 29, 1, 0, 1, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 4, 2, 41, 8, 2, 11, 2, 12, 2, 42, 1, 3, 1, 3, 1, 3, 1, 3, 1, 3, 1, 3, 1, 3, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 1, 4, 3, 4, 62, 8, 4, 1, 5, 1, 5, 3, 5, 66, 8, 5, 1, 6, 1, 6, 1, 6, 0, 0, 7, 0, 2, 4, 6, 8, 10, 12, 0, 1, 2, 0, 2, 4, 7, 8, 68, 0, 17, 1, 0, 0, 0, 2, 33, 1, 0, 0, 0, 4, 40, 1, 0, 0, 0, 6, 44, 1, 0, 0, 0, 8, 51, 1, 0, 0, 0, 10, 65, 1, 0, 0, 0, 12, 67, 1, 0, 0, 0, 14, 16, 3, 2, 1, 0, 15, 14, 1, 0, 0, 0, 16, 19, 1, 0, 0, 0, 17, 15, 1, 0, 0, 0, 17, 18, 1, 0, 0, 0, 18, 23, 1, 0, 0, 0, 19, 17, 1, 0, 0, 0, 20, 22, 3, 6, 3, 0, 21, 20, 1, 0, 0, 0, 22, 25, 1, 0, 0, 0, 23, 21, 1, 0, 0, 0, 23, 24, 1, 0, 0, 0, 24, 27, 1, 0, 0, 0, 25, 23, 1, 0, 0, 0, 26, 28, 3, 8, 4, 0, 27, 26, 1, 0, 0, 0, 28, 29, 1, 0, 0, 0, 29, 27, 1, 0, 0, 0, 29, 30, 1, 0, 0, 0, 30, 31, 1, 0, 0, 0, 31, 32, 5, 0, 0, 1, 32, 1, 1, 0, 0, 0, 33, 34, 5, 6, 0, 0, 34, 35, 5, 13, 0, 0, 35, 36, 5, 10, 0, 0, 36, 37, 3, 4, 2, 0, 37, 38, 5, 11, 0, 0, 38, 3, 1, 0, 0, 0, 39, 41, 5, 13, 0, 0, 40, 39, 1, 0, 0, 0, 41, 42, 1, 0, 0, 0, 42, 40, 1, 0, 0, 0, 42, 43, 1, 0, 0, 0, 43, 5, 1, 0, 0, 0, 44, 45, 5, 5, 0, 0, 45, 46, 5, 13, 0, 0, 46, 47, 5, 1, 0, 0, 47, 48, 3, 10, 5, 0, 48, 49, 5, 9, 0, 0, 49, 50, 5, 13, 0, 0, 50, 7, 1, 0, 0, 0, 51, 52, 5, 12, 0, 0, 52, 53, 5, 13, 0, 0, 53, 54, 5, 1, 0, 0, 54, 55, 3, 10, 5, 0, 55, 56, 5, 9, 0, 0, 56, 61, 5, 13, 0, 0, 57, 58, 5, 10, 0, 0, 58, 59, 3, 4, 2, 0, 59, 60, 5, 11, 0, 0, 60, 62, 1, 0, 0, 0, 61, 57, 1, 0, 0, 0, 61, 62, 1, 0, 0, 0, 62, 9, 1, 0, 0, 0, 63, 66, 3, 12, 6, 0, 64, 66, 5, 13, 0, 0, 65, 63, 1, 0, 0, 0, 65, 64, 1, 0, 0, 0, 66, 11, 1, 0, 0, 0, 67, 68, 7, 0, 0, 0, 68, 13, 1, 0, 0, 0, 6, 17, 23, 29, 42, 61, 65]
```

Comment:
No changes needed just confused what this is for. If you see this, just send a message through Discord and explain to me what this is for if you can

Reply:
That is part of the ANTLR generated code. I really don't know what it is for either though I suspect it encodes some type of transition table for the parser.